src/components/App.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import '../styles/App.css';
+
+class App extends Component {
```

Comment:
I think we'll need to agree on a convention for presentation components.
This PR is currently mixing class based and functional components.

Reply:
@Karuhanga thanks for your review, may I refer you to the [story rubric](https://www.pivotaltracker.com/story/show/163383129)  for the structure, and I believe we will definitely use both class and functional components

Reply:
@Karuhanga from my understanding, one uses the functional components when not interacting with the state and class components when interacting with the state.

Reply:
Not necessarily. Hooks allow us access state from functions as well.

That said though, I have talked to @codjoero and @FahdJamy, I'd like everyone else's opinion as well, but the consensus is in the future;
- We'll stick to stateless functional components for presenters
- If a component needs state, we'll wrap it in a container

I hope the rationale for this makes sense.

Reply:
@Karuhanga yes I think we should stick with that convention